libs/ui-lib/lib/cim/components/InfraEnv/InfraEnvFormPage.tsx

```diff
@@ -78,7 +78,7 @@ const validationSchema = (usedNames: string[], t: TFunction) =>
       name: richNameValidationSchema(t, usedNames),
       location: locationValidationSchema(t).required(t('ai:Location is a required field.')),
       pullSecret: pullSecretValidationSchema.required(t('ai:Pull secret is a required field.')),
-      sshPublicKey: sshPublicKeyValidationSchema,
+      sshPublicKey: sshPublicKeyListValidationSchema,
```

Comment:
Let's keep the validation schema for a single key here. This field maps to `InfraEnv.spec.sshAuthorizedKey` and only one key is expected here.